lib/api.rb

```diff
@@ -0,0 +1,20 @@
+module Api
```

Comment:
@jrafanie since `Api::Settings` is already a thing, mind if I address this in another PR?